### PR TITLE
1 rabbit to slack code refactoring

### LINKED ISSUE #1: [rabbit-to-slack] Code refactoring
1. Find out how the app works
2. Try to run the application, you can use the script https://github.com/Kv-126-DevOps/None/issues/15#issuecomment-1079721724
3. In your opinion, add the necessary changes to the code
    - Format code according to PEP-8
    - Add check of required environment variables; set other environment variables to default values
    - Rename the environment variables according to the accepted standard naming convention
    - Fill out README.md according to the [README_service_template.md](https://github.com/Kv-126-DevOps/None/blob/main/README_service_template.md)
4. Create a pull request from the branch `issue-<ISSUE NUMBER>`

### PR DESCRIPTION
**Pull request by Olyakaya**
merge

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+RABBIT_HOST=rabbit
```
Comment (anatolek) [inline .env]: Not sure if this file is needed at all. From my perspective, it's better to specify default values right in the application code. Let me know if you have a different vision.
But in general, you can leave it like that.

File: README.md [hunk 2]
```diff
@@ -2,17 +2,34 @@
...
+	
+| Parameter     | Default     | 
+|:--------------|:------------|
+| RABBIT_HOST   |    rabbit   | 
+| RABBIT_PORT   |    5672     |
+| RABBIT_QUEUE  |    slack    |
+| SLACK_URL     | get from the selected github repo -> Webhook URL|
+	
+### Testing
+    When  json-filter and rabbit-to-slack are configured , messages about new issues in the connected Github repo will be appeared in the configured for integration slack channel.
```
Comment (anatolek) [inline README.md]:
```suggestion
    When json-filter and rabbit-to-slack are configured, messages about new issues in the connected Github repo will appear in the configured for integration slack channel.
```

File: requirements_origin.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+greenlet==1.1.1
```
Comment (anatolek) [inline requirements_origin.txt]: Please delete the original file if it is no longer needed

**Review by anatolek — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (Olyakaya) [reply .env]: This file was taken from pycharm. It doesn't needed.

File: requirements_origin.txt (hunk 3 above)
Comment (Olyakaya) [reply requirements_origin.txt]: ok

**Pull request merged**